### ISSUE TITLE
[feature request] Bot in Facebookgruppe einladen

### ISSUE THREAD

**Issue by Staazvaind** [eknoes/covidbot#167]
... und Abo dort als Beitrag posten.

Gruß

**Comment by eknoes** [comment]
Guter Vorschlag! Ich glaube, das ist leider aktuell nicht möglich: Wir haben - soweit ich das sehe - als Seite keinen Zugriff auf Gruppen, zumindest nicht via API: https://stackoverflow.com/a/49810130/1692478

Kennst du ähnliche Projekte, bei denen sowas gemacht wird? Dann könnte ich mir das dort mal ansehen.

**Comment by Staazvaind** [comment]
Wenn eine Gruppe das erlaubt, können diese Seiten zulassen. Ich bin bspw. mit meiner Projektseite "Oldenburg im Mittelalter und Renaissance" MItglied in zwei Gruppe und kann dort im Namen meiner Seite posten. Deshalb kam mir überhaupt die Idee.

Theoretisch müsste die Abofunktion nach Einladung in eine Gruppe "nur" einen Beitrag verfassen.

**Comment by eknoes** [comment]
Ich finde die Idee echt ganz cool, schaffe es aber zumindest eher noch nicht an diesem Wochenende mir das mal genauer anzusehen. Der API Call scheint relativ easy zu sein: https://developers.facebook.com/docs/groups-api/common-uses#posting-on-a-group

Dafür muss man dann den Bot hinzufügen, wie hier beschrieben: https://www.facebook.com/help/261149227954100

Die Frage ist ein bisschen, wie man das umsetzt, herauszufinden, welchen Ort die Gruppe dann abonniert hat.

**Comment by Staazvaind** [comment]
ja, gute Frage. Admin-PN evtl ?

**Comment by eknoes** [comment]
Ja, wir müssen dafür bei Facebook beantragen, dass wir bestimmte Rechte bekommen. Das Problem bei so "individueller" Kommunikation mit dem Bot ist, dass wir eigentlich auf einem Messengerunabhängigen Framework aufsetzen: Dadurch ist es eher nicht drin, bei bestimmten Fällen auf bestimmten Plattformen gewisse Sonderfälle zu implementieren. Mal sehen :)

**Comment by Staazvaind** [comment]
Dann drücke ich mal die Daumen !
Auf jeden Fall top Projekt !
Congrats !
